doc/user/content/self-managed/installation/install-on-aws.md

```diff
@@ -14,101 +19,237 @@ For testing purposes only. For testing purposes only. For testing purposes only.
+for details.
+
+### Helm 3.2.0+
+
+If you do not have Helm 3.2.0+, install. See the [Helm
+documentation](https://helm.sh/docs/intro/install/).
+
 ### AWS Kubernetes environment
 
+{{% self-managed/requirements-list %}}
+
 When operating in AWS, we recommend:
 
+- Using the `r8g`, `r7g`, and `r6g` families when running without local disk.
+
 - Using the `r7gd` and `r6gd` families of instances (and `r8gd` once available)
-  when running with local disk
+  when running with local disk (Recommended for production.  See [Operational guidelines](/self-managed/operational-guidelines/#locally-attached-nvme-storage-openebs) for more information.)
+
+See [A. Set up AWS Kubernetes environment](#a-set-up-aws-kubernetes-environment)
+for a sample setup.
 
-- Using the `r8g`, `r7g`, and `r6g` families when running without local disk
+## A. Set up AWS Kubernetes environment
 
-[//]: # "TODO: Add Terraform and non-Terraform instructions here (tabbed)."
+{{< tabs  >}}
 
-Materialize provides a [sampleTerraform
+{{< tab "Terraform" >}}
+
+Materialize provides a [sample Terraform
 module](https://github.com/MaterializeInc/terraform-aws-materialize/blob/main/README.md)
-to deploy a sample infrastructure on AWS with the following components:
+for evaluation purposes only. The module deploys a sample infrastructure on AWS
+(region `us-east-1`) with the following components:
 
 - EKS component
 - Networking component
-- Storage component
-- Database component for metadata storage
+- Storage component (S3 for blob storage)
+- Database component for metadata storage (PostgreSQL RDS)
 
-See the
-[README](https://github.com/MaterializeInc/terraform-aws-materialize/blob/main/README.md)
-for information on how to deploy the infrastructure.
+{{< warning >}}
 
-### `kubectl`
+{{< self-managed/terraform-disclaimer >}}
 
-Install `kubectl` and configure cluster access. For details, see the [Amazon EKS
-documentation](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html).
+{{< /warning >}}
 
-Configure `kubectl` to connect to your EKS cluster, replacing
-`<your-region>` with the region of your EKS cluster:
+1. Clone or download the [Materialize's sample Terraform
+   repo](https://github.com/MaterializeInc/terraform-aws-materialize).
 
-```bash
-aws eks update-kubeconfig --name materialize-cluster --region <your-region>
-```
+1. Go to the Materialize Terraform repo directory.
 
-{{< note >}}
+   ```bash
+   cd terraform-aws-materialize
+   ```
 
-The exact authentication method may vary depending on your EKS configuration.
+1. Copy the `terraform.tfvars.example` file to `terraform.tfvars`.
 
-{{< /note >}}
+   ```bash
+   cp terraform.tfvars.example terraform.tfvars
+   ```
 
-To verify, run the following command:
+1. Edit the `terraform.tfvars` file to set the values for your AWS environment.
+   In particular,
 
-```bash
-kubectl get nodes
-```
+   - set your `database_password` to a secure password.
```

Comment:
cc @bobbyiliev is this needed? could we auto-generate a password as part of the TF run?